srcpkgs/kubernetes-kind/template

```diff
@@ -1,13 +1,29 @@
 # Template file for 'kubernetes-kind'
 pkgname=kubernetes-kind
 version=0.10.0
-revision=1
+revision=2
 wrksrc="kind-${version}"
 build_style=go
+build_helper=qemu
 go_import_path="sigs.k8s.io/kind"
 short_desc="Kind is a tool for running local Kubernetes clusters using Docker"
 maintainer="Andy Cobaugh <andrew.cobaugh@gmail.com>"
 license="Apache-2.0"
 homepage="https://kind.sigs.k8s.io/"
 distfiles="https://github.com/kubernetes-sigs/kind/archive/v${version}.tar.gz"
 checksum=@e5511659e3e1735531d4845405782d869d4fb425b545ea89151fd76208a12a49
+_completions="bash zsh fish"
+_pkgname="kind"
+
+post_build() {
+	local cli=$(find $GOPATH/bin -name $_pkgname)
+	for shell in $_completions; do
+		vtargetrun $cli completion $shell > "${_pkgname}.${shell}"
+	done
+}
+
+post_install() {
+	for shell in $_completions; do
+		vcompletion "${_pkgname}.${shell}" $shell $_pkgname
+	done
+}
```

Comment:
I'd rather see `kind` used directly instead of `_pkgname`, I think.

Reply:
I did that in order to avoid errors and repetitions, but sure, I can use "kind" directly.